core/invoker/src/main/scala/whisk/core/container/ContainerPool.scala

```diff
@@ -92,6 +93,26 @@ class ContainerPool(
     }
 
     /**
+     *  Check whether we should use runc.  To do so,
+     *  1. The whisk config flag must be on.
+     *  2. Runc must be successfully accessible.  This is a failsafe in case runc is not set up correctly.
+     *     For this stage, logging shows success or failure if we get this far.
+     */
+    def checkRuncAccess(useRunc: Boolean): Boolean = {
+        if (useRunc) {
+            implicit val tid = TransactionId.invokerNanny
+            val (code, result) = RuncUtils.list()
+            val success = (code == 0)
+            info(this, if (success) s"Using runc. list result: ${result}"
+                               else s"Not using runc due to error (code = ${code}): ${result}")
```

Comment:
hmm this should be a warning, not info. we want to use a feature but we can't so the system does not behave as desired.